ui/packages/catalog-ui-search/src/main/webapp/component/search-interactions/search-interactions.view.js

```diff
@@ -42,8 +45,20 @@ module.exports = Marionette.LayoutView.extend({
     onRender: function(){
         this.listenTo(this.model, 'change:type', this.triggerCloseDropdown);
         this.generateSearchFormSelector();
+        if(properties.hasExperimentalEnabled()) { 
+            this.generateResultFormSelector() 
+        }
         this.generateSearchSettings();
     },
+    generateResultFormSelector: function() {
+        this.resultType.show(new ResultFormSelectorDropdownView({
+            model: new DropdownModel(),
+            modelForComponent: this.model,
+            selectionInterface: this.options.selectionInterface
```

Comment:
✏️ Doesn't look like this is used inside the `ResultFormSelectorDropdownView` at all